topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
+
+### Behaviour
+
+1. Open the combo-box menu by clicking the arrow button on the right or pressing the <shortcut>Down</shortcut> arrow key when the combo box is focused.
 The menu opens down by default. If there is not enough space, the menu opens up.
 
-When the menu opens, select the element that was shown in the closed combo box. If a custom value is entered, then do not select a value in the list.
+2. When the menu opens, select the element that was shown in the closed combo-box. If a custom value is entered, then do not select a value in the list.
 
-Move the selection in the menu and update the value in the combo box by pressing the <control>Up</control> and <control>Down</control> arrow keys.
+3. Move the selection in the menu and update the value in the combo-box by pressing the <control>Up</control> and <control>Down</control> arrow keys.
 On mouse hover, move the selection to an item the cursor is pointing to and update the value by clicking the mouse button or pressing <shortcut>Enter</shortcut>.
 
-The menu remains opened until the user clicks the item in the list, presses <shortcut>Enter</shortcut> or <shortcut>Esc</shortcut>, clicks outside the menu,
+4. The menu remains opened until the user clicks the item in the list, presses <shortcut>Enter</shortcut> or <shortcut>Esc</shortcut>, clicks outside the menu,
 or switches to another app.
 
-#### Menu items
+### Menu items
 
-The menu list contains predefined or the most likely options. Follow the rules for [drop-down menu items](drop_down.md#menu-items).
+* The menu list contains predefined or the most likely options. Follow the rules for [drop-down menu items](drop_down.md#menu-items).
 
-If the user needs to return to previous inputs, add such values to the end of the list when clicking the confirmation button in the dialog.
+* If the user needs to return to previous inputs, add such values to the end of the list when clicking the confirmation button in the dialog:
 
-![](maven.png){width=413}
+  ![A combo-box with previously entered values in the menu](combobox_menu_items_1.png){width=706}
 
-Use [built-in buttons](built_in_button.md) to add values or expand the combo box, e.g., the browse button:
+* Use [built-in buttons](built_in_button.md) to add values or expand the combo-box, e.g., the browse button:
 
-![](built_in_button.png){width=384}
+  ![A combo-box with the browse built-in button](combobox_menu_items_2.png){width=706}
 
 ## Validation
 
-If the user enters an invalid value, highlight the combo box with red and show an error message in a tooltip. For
-more details, see [Validation errors](validation_errors.md).
+If the user enters an invalid value, highlight the combo-box with red and show an error message in a tooltip. For
+more details, see [validation errors](validation_errors.md).
 
-![](validation.png){width=235}
+![A combo-box with an error message](combobox_validation.png){width=706}
 
 ## Sizes and placement
```

Comment:
We'll need to remove sizes instructions, so I suggest renaming this to "How to layout" and give a link to https://plugins.jetbrains.com/docs/intellij/layout.html#labeled-input-controls